strings/tips.txt

```diff
@@ -188,9 +184,8 @@ As a Nuclear Operative, you might end up in a situation where the AI has bolted
 As a Nuclear Operative, buy tools to heal your robots. There's a engi-vend in the backroom!
 As a Monkey, you can crawl through air or scrubber vents by alt+left clicking them. You must drop everything you are wearing and holding to do this, however.
 As a Monkey, you can still wear a few human items, such as backpacks, gas masks and hats, and still have two free hands.
-As the Malfunctioning AI, you can shunt to an APC if the situation gets bad. This disables your doomsday device if it is active.
+As the Malfunctioning AI, you can shunt to an APC if the situation gets bad. This disables your doomsday device.
```

Comment:
pretty sure you don't have to be going nuclear to shunt

Reply:
If you shunt you lose the ability to doomsday at all, I believe

Reply:
The original phrased it like it would only disable doomsday if it was active at the time you shunted.